.github/workflows/job_vlm_sample_llava.yml

```
@@ -13,6 +13,10 @@ on:
 env:
   l_u22_ov_link: https://storage.openvinotoolkit.org/repositories/openvino/packages/nightly/2025.1.0-17911-83c047443de/l_openvino_toolkit_ubuntu22_2025.1.0.dev20250116_x86_64.tgz
 
+
+permissions: read-all
```

```suggestion
permissions: read-all  # Required by https://github.com/ossf/scorecard/blob/e23b8ad91fd6a64a0a971ca4fc0a4d1650725615/docs/checks.md#token-permissions
```
To stay aligned with the rest of wirkflows